httpgrpc/httpgrpc.go

```diff
@@ -3,11 +3,10 @@ package httpgrpc
 import (
 	"fmt"
 
-	"github.com/golang/protobuf/ptypes"
-	"github.com/golang/protobuf/ptypes/any"
+	spb "github.com/gogo/googleapis/google/rpc"
+	ptypes "github.com/gogo/protobuf/types"
```

Comment:
Whats the reason for the import rename?

Reply:
So I didn't have to edit the lines that said `ptypes.MarshalAny` and `ptypes.UnmarshalAny`

Reply:
Oh okay; I'd personally just change these lines, but not a big deal.

Reply:
changed